pandas/tests/groupby/test_groupby.py

```diff
@@ -361,6 +363,119 @@ def f3(x):
         df2.groupby("a").apply(f3)
 
 
+def test_groupby_indices_error():
+    # GH 26860
+    # Test if DataFrame Groupby builds gb.indices
+    dt = pd.to_datetime(["2018-01-01", "2018-02-01", "2018-03-01"])
+    df = DataFrame(
+        {
+            "a": Series(list("abc")),
+            "b": Series(dt, dtype="category"),
+            "c": pd.Categorical.from_codes([-1, 0, 1], categories=[0, 1]),
+        }
+    )
+
+    df.groupby(["a", "b"]).indices
+
+
+@pytest.mark.parametrize(
+    "gb_cols",
+    [
+        "int_series",
```

Comment:
personal preference: i'd drop the series from these names

Reply:
Sure thing.